packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionWorkflowGraph.tsx

```diff
@@ -52,9 +76,15 @@ export const ExecutionWorkflowGraph: React.FC<ExecutionWorkflowGraphProps> = ({
 
   const onCloseDetailsPanel = () => setSelectedNodes([]);
 
+  const [selectedMapTask, setSelectedMapTask] = useState<ExternalResource[] | null>(null);
+  const onMapTaskSelectionChanged = (newSelection: ExternalResource[] | null) => {
+    setSelectedMapTask(newSelection);
```

Comment:
nit: this seems to be a plain wrapper around setSelectedMapTask, you could just pass it directly and avoid an extra reference